jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/spec/client/webtarget/JAXRSClientIT.java

```diff
@@ -44,10 +44,8 @@
 public class JAXRSClientIT extends JAXRSCommonClient {
 
   public JAXRSClientIT() {
-    setup();
```

Comment:
The setup() is not called in this class?

Reply:
Same with this one, it doesn't appear to be an Arquillian test.